ext/curl/tests/curl_pushfunction.phpt

```diff
@@ -0,0 +1,62 @@
+--TEST--
+Test CURLMOPT_PUSHFUNCTION
+--CREDITS--
+Davey Shafik
+Kévin Dunglas
+--EXTENSIONS--
+curl
+--SKIPIF--
+<?php
+include 'skipif-nocaddy.inc';
+
+$curl_version = curl_version();
+if ($curl_version['version_number'] < 0x073d00) {
+    exit("skip: test may crash with curl < 7.61.0");
```

Comment:
Why "skip:" here? (colon)

Reply:
I just copied the message that was already existing in similar tests: https://github.com/search?q=repo%3Aphp%2Fphp-src+skip%3A+test+may+crash+with+curl+%3C+7.61.0&type=code